core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -451,6 +451,25 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .build();
 
+  //
+  // Diagnostic logs
+  //
+  public static final PropertyKey DIAGNOSTIC_LOG_INTERVAL_MS =
+      new Builder(Name.DIAGNOSTIC_LOG_INTERVAL_MS)
+          .setDescription("Period between two consecutive diagnostic executions.")
+          .setDefaultValue("86400000")
+          .setIsHidden(true)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey DIAGNOSTIC_LOG_LEVEL = new Builder(Name.DIAGNOSTIC_LOG_LEVEL)
+      .setDescription(
+          "The log level for gathering diagnostic information. The options are "
+              + "[NONE, PLAIN]")
```

Comment:
What do these levels mean? Are these somehow related to the standard log levels like `DEBUG`, `INFO`, etc?